etc/docs/site/usage-docs/Serialization-of-data.md

```diff
@@ -26,6 +26,7 @@ DataFrames from R, Pandas and Spark are serialized into the following structure:
 ```javascript
 {
   columns: [], //array of column names
+  column_types: [], //array of column type names
```

Comment:
Can the serialization use camelCase?   It's a bit awkward to have the underscore names mixed in our JS code.